src/retrace/hooks/config.py

```diff
@@ -42,11 +42,17 @@ def load_hook_config() -> Dict[str, str]:
     main_hook_config_file = Path(MAIN_CONFIG_PATH, MAIN_HOOK_CONFIG_FILE)
     hook_configs.append(main_hook_config_file)
 
-    if USER_HOOK_CONFIGS_PATH.exists():
-        hook_configs += get_config_files(USER_HOOK_CONFIGS_PATH)
-
-    if USER_CONFIG_PATH.exists():
-        hook_configs.append(USER_CONFIG_PATH)
+    try:
+        if USER_HOOK_CONFIGS_PATH.exists():
+            hook_configs += get_config_files(USER_HOOK_CONFIGS_PATH)
+    except Exception as ex:
+        log_warn("USER_HOOK_CONFIGS_PATH %s does not exist: %s" % (USER_HOOK_CONFIGS_PATH, ex))
```

Comment:
I think we prefer the "new" format strings in new code:
```suggestion
        log_warn("USER_HOOK_CONFIGS_PATH {USER_HOOK_CONFIGS_PATH} does not exist: {ex}")
```

Reply:
@mgrabovsky ok is that the only issue you see?
I made this change and it looks ok but right now my test machine is down and normally I prefer to build/test before I submit.  In this case if you don't see anything else that needs changed, are you ok with me just re-pushing this?

Reply:
Yes, it seems fine otherwise. You can push right away.